projection/bridge_activity/bridge_pending_activity/bridge_pending_activity.go

```diff
@@ -0,0 +1,352 @@
+package bridge_pending_activity
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/crypto-com/chain-indexing/projection/bridge_activity/types"
+
+	"github.com/crypto-com/chain-indexing/appinterface/projection/rdbprojectionbase"
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+	event_entity "github.com/crypto-com/chain-indexing/entity/event"
+	entity_projection "github.com/crypto-com/chain-indexing/entity/projection"
+	applogger "github.com/crypto-com/chain-indexing/internal/logger"
+	"github.com/crypto-com/chain-indexing/internal/primptr"
+	"github.com/crypto-com/chain-indexing/internal/utctime"
+	bridge_pending_activity_view "github.com/crypto-com/chain-indexing/projection/bridge_activity/view"
+	"github.com/crypto-com/chain-indexing/usecase/coin"
+	event_usecase "github.com/crypto-com/chain-indexing/usecase/event"
+)
+
+var _ entity_projection.Projection = &BridgePendingActivity{}
+
+type BridgePendingActivity struct {
+	*rdbprojectionbase.Base
+
+	rdbConn rdb.Conn
+	logger  applogger.Logger
+}
+
+type Config struct {
+	ThisChainId           string `mapstructure:"this_chain_id"`
+	ThisChainName         string `mapstructure:"this_chain_name"`
+	CounterpartyChainName string `mapstructure:"counterparty_chain_name"`
+	ChannelId             string `mapstructure:"channel_id"`
+	StartingHeight        int64  `mapstructure:"starting_height"`
+}
+
+func NewBridgePendingActivity(logger applogger.Logger, rdbConn rdb.Conn) *BridgePendingActivity {
+	var config Config
+
+	return &BridgePendingActivity{
+		rdbprojectionbase.NewRDbBaseWithOptions(
+			rdbConn.ToHandle(), "BridgePendingActivity", rdbprojectionbase.Options{
+				MaybeTable:     nil,
+				MaybeConfigPtr: &config,
+			}),
+
+		rdbConn,
+		logger,
+	}
+}
+
+func (_ *BridgePendingActivity) GetEventsToListen() []string {
+	return []string{
+		event_usecase.BLOCK_CREATED,
+
+		event_usecase.MSG_IBC_TRANSFER_TRANSFER_CREATED,
+		event_usecase.MSG_IBC_TRANSFER_TRANSFER_FAILED,
+		event_usecase.MSG_IBC_RECV_PACKET_CREATED,
+		event_usecase.MSG_IBC_ACKNOWLEDGEMENT_CREATED,
+		event_usecase.MSG_IBC_TIMEOUT,
```

Comment:
I think it should be `event_usecase.MSG_IBC_TIMEOUT_CREATED`. The current implementation will miss the timeout event.